project/Settings.scala

```diff
@@ -138,12 +198,16 @@ object Settings extends Build {
     }
   )
 
-  /* By default, assembly is not enabled for the demos module, but it can be enabled with
-    `-Ddemos.assembly=true`. From the command line this would be:
-     sbt -Ddemos.assembly=true twitter/assembly */
-  lazy val demoAssemblySettings =
-    if (System.getProperty("demos.assembly", "true").toBoolean) sbtAssemblySettings
-    else Seq.empty
+  lazy val theme = PromptTheme(List(
```

Comment:
This doesn't really respect default terminal colors so on my screen it's unreadable

![image](https://cloud.githubusercontent.com/assets/413025/6309166/4c5e6ff6-b8ff-11e4-9cfd-42e99673612e.png)